item.go

```diff
@@ -114,45 +181,67 @@ func (api *API) ItemsCreate(items Items) (err error) {
 	result := response.Result.(map[string]interface{})
 	itemids := result["itemids"].([]interface{})
 	for i, id := range itemids {
-		items[i].ItemId = id.(string)
+		items[i].ItemID = id.(string)
 	}
 	return
 }
 
-// Wrapper for item.delete: https://www.zabbix.com/documentation/2.2/manual/appendix/api/item/delete
+// ItemsUpdate Wrapper for item.update
+// https://www.zabbix.com/documentation/3.2/manual/api/reference/item/update
+func (api *API) ItemsUpdate(items Items) (err error) {
+	_, err = api.CallWithError("item.update", items)
+	return
+}
+
+// ItemsDelete Wrapper for item.delete
 // Cleans ItemId in all items elements if call succeed.
+// https://www.zabbix.com/documentation/3.2/manual/api/reference/item/delete
 func (api *API) ItemsDelete(items Items) (err error) {
 	ids := make([]string, len(items))
 	for i, item := range items {
-		ids[i] = item.ItemId
+		ids[i] = item.ItemID
 	}
 
 	err = api.ItemsDeleteByIds(ids)
 	if err == nil {
 		for i := range items {
-			items[i].ItemId = ""
+			items[i].ItemID = ""
 		}
 	}
 	return
 }
 
-// Wrapper for item.delete: https://www.zabbix.com/documentation/2.2/manual/appendix/api/item/delete
+// ItemsDeleteByIds Wrapper for item.delete
+// https://www.zabbix.com/documentation/3.2/manual/api/reference/item/delete
 func (api *API) ItemsDeleteByIds(ids []string) (err error) {
+	deleteIds, err := api.ItemsDeleteID(ids)
+	if err != nil {
+		return
+	}
+	l := len(deleteIds)
+	if len(ids) != l {
+		err = &ExpectedMore{len(ids), l}
+	}
+	return
+}
+
+// ItemsDeleteID Wrapper for item.delete
+// Delete the item and return the id of the deleted item
+func (api *API) ItemsDeleteID(ids []string) (itemids []interface{}, err error) {
```

Comment:
Why have both `ItemsDeleteByIds()` and `ItemsDeleteID()` functions?

As `ItemsDeleteID()` also accepts a list of ids, it should be named `ItemsDeleteIDs()` if really needed.